packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -220,9 +221,13 @@ class UsersProfileClass extends Component<UsersProfileProps,UsersProfileState> {
     const currentShowLowKarma = (parseInt(query.karmaThreshold) !== DEFAULT_LOW_KARMA_THRESHOLD)
     
     const username = Users.getDisplayName(user)
+    const metaDescription = `${username}'s profile on ${siteNameWithArticleSetting.get()} — ${taglineSetting.get()}`
```

Comment:
This seems generally useful, are you also submitting upstream?

Reply:
Yes.